docs/sources/mimir/visualize/grafana.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Visualize your metric data with Grafana
+title: Visualize metric data
+menuTitle: Visualize data
+weight: 45
+keywords:
+  - visualize
+  - grafana
+  - dashboards
+---
+
+# Visualize metric data
+
+[Grafana](/grafana/download/) has built-in support for Mimir through the Prometheus data source type.
+
+## Before you begin
+
+1. Log in to your Grafana instance.
+   If this is your first time running Grafana,
+   the username and password are both `admin`.
+1. From within Grafana, go to `Connections` > `Add new connection`.
+1. Search for Prometheus, and select **Create a Prometheus data source**.
+1. In the **HTTP** > **Prometheus server URL** field, enter a server URL:
+
+   - If you deployed Mimir via the `mimir-distributed` Helm chart,
```

Comment:
I would also mention that both `helm install` and `helm upgrade` print the actual URLs to use for reading/writing to Mimir.

Reply:
Thank you. This is super helpful for the user.